capstone/src/main/java/com/google/sps/servlets/UpdateVotesServlet.java

```diff
@@ -0,0 +1,75 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.KeyFactory;
+
+@WebServlet("/update-votes")
+public class UpdateVotesServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get id of changed checkbox
+    String optionIdString = request.getParameter("id");
+    // Get whether checkbox is currently checked
+    String checked = request.getParameter("checked");
```

Comment:
Can you rename this to something along the lines of isOptionChecked? It's pretty ambiguous otherwise as to what checked means. 

Also, can you use Boolean's parseBoolean or valueOf methods to convert this to a boolean/Boolean? Checking if a string === "false"/"true" is not pretty.
https://www.javatpoint.com/java-string-to-boolean